Assets/00_Content/Scripts/Rounds/RoundController.cs

```diff
@@ -25,11 +31,13 @@ public class RoundController : MonoBehaviour {
 		private void Update() {
 			roundTimer = Mathf.Max(0, roundTimer - Time.deltaTime);
 
-			if (roundTimer <= 0) {
-				currentRound++;
-				SendNextDifficulty();
-				roundTimer = roundDuration;
-			}
+			if (roundTimer <= 0) RoundWon();
```

Comment:
Would this be a fitting place to stop the viking controller spawning in more vikings?

Reply:
Fixed.